start-up/moderation-tutorial.md

```diff
@@ -1,54 +1,55 @@
 # Moderation Tutorial
 
-## Intro
+This guide will give you the basic information needed to moderate your Discord server with MoonlightBot. It includes a list of commands and how they work, complete with their options, infractions, and log information to help you better understand how to use them
 
-This guide will give you the basic information needed to moderate your Discord server with MoonlightBot. It includes a list of commands and how they work, complete with their arguments, infractions, and log information to help you better understand how to use them
+## Command Options
 
-## Arguments
+These are options you may use while executing moderation commands. You can find a more in-depth guide about options in the [arguments page](/start-up/arguments.md)
 
-Arguments are options you may use while executing moderation commands. You can find a more in-depth guide about arguments in the [arguments page](/start-up/arguments.md)
+![A list of example options in a mute command](/.gitbook/assets/MuteArguments.png "Options Example")
 
-![](/.gitbook/assets/MuteArguments.png)
-
-- **`user`**: The target member of the command
-- **`reason`**: The reason for which you are executing the command
-- **`notify`**: Whether or not the bot will message the user after the command is executed
-- **`no-infraction`**: Whether or not the command will count towards the user's infractions
+- `user`: The user to execute the moderation action on
+- `reason`: The reason for which you are taking the action, useful for record keeping. The reason shows up in infractions, logs, and notifications from the bot
+- `notify`: Whether or not the bot will message the user after the command is executed, as shown below
+![An example of a notification from the bot](/.gitbook/assets/NotifyExample.png "Notify Example")
+- `no-infraction`: Whether or not the command will count towards the user's infractions
 
 {% hint style="info" %}
-If both the `notify` and `no-infraction` arguments are true, the target will not be notified in order to prevent harrassment via MoonlightBot
+If both the `notify` and `no-infraction` options are true, the target will not be notified in order to prevent harassment via MoonlightBot
 {% endhint %}
 
-- **`duration`**: The amount of time for which the user will be affected by the command. For more information about this argument, refer to the [arguments page](/start-up/arguments.md#durations)
-- **`remove-roles`**: Exclusive to the mute commands, this argument will remove all of the user's roles except the muterole if true
-- **`role`**: The role to assign/remove in a role command
+- `duration`: The amount of time for which the user will be affected by the effects of the command. You can use any duration ranging from weeks to milliseconds. For example, `1w2d` and `1 week 2 days` will both affect the target of the command for 1 week and 2 days.
```

Comment:
Leave this explanation to the page with a link how it was before, i was more referring to "Bor more information about this argument" being too generic